spec/functions/validate_domain_name_spec.rb

```diff
@@ -5,7 +5,6 @@
 describe 'validate_domain_name' do
   describe 'signature validation' do
     it { is_expected.not_to eq(nil) }
-    it { is_expected.to run.with_params.and_raise_error(Puppet::ParseError, %r{wrong number of arguments}i) }
```

Comment:
This is a change: before calling the function without parameters caused an exception, but now it pass (I would have assumed `repeated_param` was one or more but it seems to allow 0 values).

While we can add some code to make match the old behavior, it seems overkill and I decided to just ignore this corner case.

Reply:
I addressed this in #1351.